cmds/vpdbootmanager/addentry/netboot.go

```diff
@@ -0,0 +1,56 @@
+package addentry
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/booter"
+	"github.com/systemboot/systemboot/pkg/vpd"
+)
+
+func addNetbootEntry() {
+	if len(os.Args) < 5 {
```

Comment:
I'd rather pass arguments to `addNetbootEntry`, so it can work in isolation instead of depending on the external environment